modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/DeltaFilePageStoreIo.java

```diff
@@ -110,4 +149,21 @@ public long pageOffset(long pageId) {
     public int fileIndex() {
         return header.index();
     }
+
+    /**
+     * Returns page offsets within the store file.
+     */
+    public long[] pageOffsets() {
```

Comment:
why copy int array to a long array? Seems a bit like overkill

Reply:
Fix it